### PR TITLE
Pow

### PR DESCRIPTION
**Pull request by asutula**
Integrating powergate grpc calls that are supported by Hub. Still needs some integration tests, but let's get some eyes on this and get a pre-release out.

### CODE REVIEW

**Comment by andrewxhill** [comment]
hey @asutula is there something simple blocking here?

**Comment by andrewxhill** [comment]
we need to do a draft release so staging is usable, wondering if we add this in

File: packages/pow/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+**Install**
+
+```js
+npm install --save git@github.com:textileio/js-hub.git
+```
+
+**Usage**
+
+```js
+import { Buckets } from '@textile/hub'
```
Comment (andrewxhill) [inline packages/pow/README.md:16]: copy/pasta

File: packages/pow/src/pow.ts [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+  newAddr,
+  peers,
+  sendFil,
+  show,
+  showAll,
+} from './api'
+
+const logger = log.getLogger('users')
+
+export class Pow extends GrpcAuthentication {
```
Comment (andrewxhill) [inline packages/pow/src/pow.ts:41]: I think at minimum all the public methods need a doc string. ideally at least 1 or 2 examples for the top level Pow class for now (i'll open a ticket for creating an example for the other public methods later)

You can see an example of what this looks like here,

https://textileio.github.io/js-hub/docs/hub.buckets

The Buckets class is documented with 3 examples, those feel like an example or a few good minimum to give someone an idea of how to get started. 

The table below, "Methods" shows the list of all public methods with the one line description of the method, that's why i think those one-liners are a minimum for each of the new methods

**Review by andrewxhill — APPROVED** [review]

File: packages/pow/README.md (hunk 1 above)
Comment (asutula) [reply packages/pow/README.md:16]: done

File: packages/pow/src/pow.ts (hunk 2 above)
Comment (asutula) [reply packages/pow/src/pow.ts:41]: Ok got the descriptions and params in for the public methods. Will address https://github.com/textileio/js-hub/issues/489 later.

**Pull request merged**